eslint.config.js

```diff
@@ -0,0 +1,16 @@
+const txoConfig = require('eslint-config-txo-typescript')
+
+/** @type {import('eslint').Linter.FlatConfig[]} */
+const config = [
+  ...txoConfig.default,
+  {
+    files: ['sample/**/*.ts'],
+    languageOptions: {
+      parserOptions: {
+        project: './__tests__/tsconfig.json',
```

Comment:
why ?

Reply:
because `./__tests__/tsconfig.json` has this include:
https://github.com/technology-studio/redux/blob/c66cfac0de026e7a0c1c73f2e564a3bdcb5f11e6/__tests__/tsconfig.json#L4-L7

Reply:
because `./__tests__/tsconfig.json` has this include:
https://github.com/technology-studio/redux/blob/c66cfac0de026e7a0c1c73f2e564a3bdcb5f11e6/__tests__/tsconfig.json#L4-L7

Reply:
I updated `__tests__/tsconfig.json` according to https://github.com/technology-studio/test-boilerplate-typescript/blob/auto-update-dependencies/__tests__/tsconfig.json